sway/ipc-server.c

```diff
@@ -56,6 +57,13 @@ void ipc_client_disconnect(struct ipc_client *client);
 void ipc_client_handle_command(struct ipc_client *client);
 bool ipc_send_reply(struct ipc_client *client, const char *payload, uint32_t payload_length);
 
+static void handle_display_destroy(struct wl_listener *listener, void *data) {
+	if (ipc_event_source) {
+		wl_event_source_remove(ipc_event_source);
+		ipc_event_source = NULL;
+	}
```

Comment:
We should also remove the `ipc_display_destroy` listener

Reply:
Good point. I'll do that at the same time as the rest if I get an answer to my question at the end of original post